tests/end_to_end/function-pointer2/irix-o2-out.c

```diff
@@ -1,3 +1,6 @@
+extern int (float) bar;
+extern int (*)(float) glob2;
```

Comment:
`glob2`'s type is right here; the formatting of `bar` is wrong but that gets fixed by #106

Reply:
and `glob2`'s formatting too I imagine